citro3d/src/math/matrix.rs

```diff
@@ -215,7 +182,33 @@ impl Matrix4 {
                 camera_up.0,
                 coordinates.is_left_handed(),
             );
-            Self::new(out.assume_init())
+            Self::from_raw(out.assume_init())
         }
     }
 }
+
+impl core::fmt::Debug for Matrix4 {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_tuple("Matrix4").field(&self.rows_wzyx()).finish()
+    }
+}
+impl PartialEq<Matrix4> for Matrix4 {
+    fn eq(&self, other: &Matrix4) -> bool {
+        self.rows_wzyx() == other.rows_wzyx()
+    }
+}
+impl Eq for Matrix4 {}
```

Comment:
yeah I think I moved them to here because I found it weird they were in a separate module and when reimplementing them automatically just put them alongside